dev/tasks/tasks.yml

```diff
@@ -1098,6 +1098,15 @@ tasks:
         UBUNTU: 20.04
       image: ubuntu-cpp-bundled
 
+  test-ubuntu-24.04-cpp-bundled-offline:
+    ci: github
+    template: docker-tests/github.linux.yml
+    params:
+      env:
+        UBUNTU: 24.04
+      image: ubuntu-cpp-bundled-offline
+      timeout: 120
```

Comment:
The job took 88 minutes. Should we put the timeout to `100`? `90` seems a little tight.